symphony/lib/core/class.configuration.php

```diff
@@ -28,6 +28,12 @@ class Configuration
     private $_forceLowerCase = false;
 
     /**
+     * The string representing the tab characters used to serialize the configuration
+     * @var string
+     */
+    const TAB = '    ';
```

Comment:
Yeah, I've changed it to spaces since all of our generated files and our code files are indented with 4 spaces